llvm/tools/llvm-objcopy/ObjcopyOptions.cpp

```diff
@@ -731,7 +731,15 @@ objcopy::parseObjcopyOptions(ArrayRef<const char *> RawArgsArr,
         llvm::crc32(arrayRefFromStringRef(Debug->getBuffer()));
   }
   Config.SplitDWO = InputArgs.getLastArgValue(OBJCOPY_split_dwo);
+
   Config.SymbolsPrefix = InputArgs.getLastArgValue(OBJCOPY_prefix_symbols);
+  Config.SymbolsPrefixRemove =
+      InputArgs.getLastArgValue(OBJCOPY_prefix_symbols_remove);
+  if (!Config.SymbolsPrefix.empty() && !Config.SymbolsPrefixRemove.empty())
+    return createStringError(
+        errc::invalid_argument,
+        "--prefix-symbols and --prefix-symbols-remove are mutualy exclusive");
```

Comment:
Why this error? It seems to me like you could do the prefix removal first and then the addition of the prefix, e.g. with removing "foo" and adding "bar", you could change "fooblob" into "barblob" with one llvm-objcopy run.

Reply:
Because it can be error prone. We do not specify the ordering of adding prefixes and removing prefix. The results can be different with a different order. If the user intentionally needs both actions, they can call llvm-objcopy twice.

Reply:
Yeah it is better to not facilitate both options at the same time. Picking the last flag could be another option but erroring out is also fine.

Reply:
Sorry, I disagree. There is a standard model for handling multiple options in llvm-objcopy (e.g. `--remove-section` and then `--add-section`) which is to perform the subtractive option first and then do the additive one, regardless of the order in which they appear on the command-line, and this isn't significantly different to that style. In other words, in this case you'd apply the (either last value or all, I'd actually consider doing all, in order) prefix remove option before doing the prefix add option. You can mention this in the documentation and enforce the ordering by tests. Honestly, I don't think there's any risk of confusion either, since it's fairly obvious what has happened by simply looking at the names.

Reply:
Done. Added test to enforce the ordering.